src/common/utility.c

```diff
@@ -97,6 +97,28 @@ guint dt_util_str_occurence(const gchar *haystack, const gchar *needle)
   return o;
 }
 
+gchar *dt_util_float_to_str(const gchar *format, const double value)
+{
+#if defined(WIN32)
+  _configthreadlocale(_ENABLE_PER_THREAD_LOCALE);
+  setlocale (LC_NUMERIC, "C");
+#else
+  locale_t locale = uselocale((locale_t) 0);
```

Comment:
Why not simply:

```
  locale_t nlocale = newlocale(LC_NUMERIC_MASK, "C", (locale_t) 0);
  locale_t locale = uselocale(nlocale);
```

Reply:
Indeed, it saves a function call.

It's hard to get you satisfied 😉
Your review work is incredible good!

Reply:
Thanks for the nice comment @zisoft, yet I'm missing quite some issues :) It is hard to spot everything.